src/reference/asciidoc/webflux.adoc

```diff
@@ -319,3 +319,9 @@ See <<./http.adoc#http-outbound,HTTP Outbound Components>> for more possible con
 
 Since WebFlux components are fully based on the HTTP protocol, there is no difference in the HTTP headers mapping.
 See <<./http.adoc#http-header-mapping,HTTP Header Mappings>> for more possible options and components to use for mapping headers.
+
+[[webflux-request-attributes]]
+=== WebFlux Request Attributes
+
+Starting with version 6.0, Webflux outbound request `WebFluxRequestExecutingMessageHandler` can have request attributes which can be specified in `setAttributeVariablesExpression()`.
+This will take an `expression` that should be evaluated in `Map`.
```

Comment:
I still would like to see a general sentence for reasoning behind these attributes.
Doesn't look like this is a common feature since we haven't bumped to its missing support for since version `5.0`.
Probably the security would be a great sample to carry on some info from message to the request where downstream filter would expect an attribute to be able to authorize a principal or so.

My point is that you need to convince me somehow in using this new feature from the doc.
No one is going to look into those tests in the project.

Reply:
how about this?

Starting with version 6.0, Webflux outbound request `WebFluxRequestExecutingMessageHandler` can have request attributes which can be specified in `setAttributeVariablesExpression()`.
This will take an `expression` that should be evaluated in `Map`.
This will be helpful if an information in form of key-value object needs be passed from Message to request and downstream filter will get access to this attribute for further processing.

Reply:
Sounds good!
👍